.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
```

Comment:
I guess this is https://pre-commit.com/ ? This looks interesting, but out of scope for this PR / issue. Let's take this out.

Reply:
Yes, I added it because I thought we might want to add a pre-commit hook with these checks as well. But yes, you're right that should be part of a separate issue/PR.